<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BilateralFiltering</class>
 <widget class="QMainWindow" name="BilateralFiltering">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BilateralFiltering</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="BilateralFilterWidget" name="bilateralFilterWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>560</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bilateral filter</string>
    </property>
   </widget>
   <widget class="QSlider" name="locationParamSlider">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>722</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="colorParamSlider">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>752</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>15</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>721</y>
      <width>134</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Location parameter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>751</y>
      <width>112</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Color parameter</string>
    </property>
   </widget>
   <widget class="QLabel" name="locationParamLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>720</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="colorParamLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>750</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadImageButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load image</string>
    </property>
   </widget>
   <widget class="QLabel" name="messageLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>790</y>
      <width>591</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="originalImageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>560</width>
      <height>250</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Original image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BilateralFilterWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>bilateralfilterwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
